src/cdk/table/table.ts

```diff
@@ -423,6 +436,8 @@ export class CdkTable<T> implements AfterContentChecked, CollectionViewer, OnDes
       protected readonly _differs: IterableDiffers,
       protected readonly _changeDetectorRef: ChangeDetectorRef,
       protected readonly _elementRef: ElementRef, @Attribute('role') role: string,
+      @Inject(_VIEW_REPEATER_STRATEGY)
+      protected readonly _viewRepeater: ViewRepeater<T, RenderRow<T>, RowContext<T>>,
```

Comment:
We generally treat constructor changes like this as a breaking change because someone might be extending the class. We'd need to make this the last argument and make it optional

Reply:
Done